op-deployer/book/src/reference-guide/custom-superchains.md

```diff
@@ -0,0 +1,157 @@
+# Custom Superchains
+
+While OP Deployer was designed primarily for use with the Optimism Superchain, it also supports managing chains that
+were deployed to custom Superchain deployments. This is particularly common for RaaS providers, whose customers often
+request deployments with custom L1s (or L2s, in the case of L3s) or governance. This guide will walk you through the
+process of managing these chains using OP Deployer.
+
+> [!danger]
+> Chains deployed in this way are not subject to Optimism Governance. They may be running customized or unaudited
+> code. Use at your own risk.
+
+## Bootstrapping
+
+The first step to deploying a custom Superchain is to bootstrap it onto an L1. This process will:
+
+- Deploy contract implementations that will be shared among all OP Chains deployed using this Superchain.
+- Deploy Superchain-wide management contracts like `SuperchainConfig` and `SuperchainVersions`.
+- Set up ownership so that you can control the Superchain.
+
+You will use the `bootstrap` [family][bootstrap] of commands on `op-deployer` to do this.
+
+[bootstrap]: ../user-guide/bootstrap.md
+
+### Superchain Bootstrap
+
+To begin, bootstrap the Superchain onto your chosen L1 with the following command:
```

Comment:
"bootstrap the Superchain" == deploy OPCM onto a new L1, correct? Is this doing anything else under the hood?

Reply:
oh, I see that OPCM gets deployed as part of "Bootstrapping Implementations". What's the difference between "Superchain Boostrap" and "Bootstrapping Implementations," at a very high level? Maybe we can add a sentence or two to clarify and orient.

Reply:
Superchain bootstrapping deploys the SuperchainConfig, ProtocolVersions, and the Superchain PAO. Implementations bootstraps all implementations (using create2, so if they already exist they will be reused) + the OPCM which points to the contracts deployed by `bootstrap superchain`.